ruby-challenges.rb

```diff
@@ -33,4 +50,15 @@
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
 
-# Pseudo code:
\ No newline at end of file
+# Pseudo code:
+# create a method that takes in the strings above
+def palindrome str
+    #write a conditional that determines if the word downcased is the same in reverse
+    if str.downcase == str.reverse.downcase
+        "#{str} is a palindrone"
+    else
+        "#{str} is not a palindrone"
+    end
+end
+# print the string if it is a palindrone
+p palindrome palindrome_test_case3
```

Comment:
Your logic is correct in all of these!  Just a couple of things:  you want to be sure to cover casing in #2 (ie, include AEIOU in your string).  In #1 , you don't need to say num.even? == true.  It's redundant. If it's even, it will return one thing, if not, return the other.  Just a few things to think about going forward. You're doing great overall, Danny!